src/chapters/loops/exercises.rst

```diff
@@ -1,27 +1,6 @@
 Exercises: Loops
 ================
 
-.. pull-quote::
-
-   Practice makes better. Repetition is a good thing.
-
-   Repetition is a good thing.
-
-   Repetition is a good thing.
-
-   Repetition is a good thing.
-
-WAIT!!!  Why type "Repetition is a good thing" four times when we can code a
-better result?  How about printing the phrase 100 times instead?
-
-.. sourcecode:: python
-   :linenos:
-
-   for line in range(100):
-      print("Repetition is a good thing.")
-
-Loops simplify repetitive tasks!
-
```

Comment:
Can you provide some context for why we should remove this intro?  It's a direct copy/paste from the JS curriculum, so removing it here means we should re-evaluate it's use in LC101.

Personally, I like how the content gives the reader a breath before diving into the exercises.  It also summarizes why we use loops and reviews the basic `for` syntax (for those who only skimmed the reading).

Reply:
So I removed it because I thought some students may react poorly to it. Whenever the exercises start with a joke, I would keep it shorter. Students who read the whole thing and don't find it funny may approach the exercises with a sense of apathy.

Reply:
OK.  I see your point, but I also think we need a small buffer between the end of the chapter and the start of the problems.  What's a shorter, fun thing we can use here instead?